internal/lsp/lspconv/converters.go

```diff
@@ -13,39 +13,47 @@ import (
 	"github.com/microsoft/typescript-go/internal/core"
 	"github.com/microsoft/typescript-go/internal/ls"
 	"github.com/microsoft/typescript-go/internal/lsp/lsproto"
-	"github.com/microsoft/typescript-go/internal/project"
 )
 
-type converters struct {
-	projectService   *project.Service
-	positionEncoding lsproto.PositionEncodingKind
+type ScriptInfo interface {
+	Text() string
+	LineMap() *LineMap
 }
 
-func (c *converters) toLspRange(fileName string, textRange core.TextRange) (lsproto.Range, error) {
-	scriptInfo := c.projectService.GetScriptInfo(fileName)
+type ConvertersHost interface {
```

Comment:
This could just be called `Host` or something (package is already called `lspconv`), or we could just provide the func only to `Converters`.